public/js/eventList.js

```diff
@@ -0,0 +1,36 @@
+
+document.querySelectorAll(".event .lower").forEach(function(event) {
+    event.dataset.height = event.clientHeight
+    event.style.height = 0
+})
+
+document.querySelectorAll(".event").forEach(function(event) {
+    let clicked = 0
+    event.addEventListener('click', function() {
+        clicked++
```

Comment:
Continually incrementing counter is a little messy, could change between 1 & 0 with `clicked = (clicked ? 0 : 1)`

Reply:
cool

Reply:
5fae0f8